pyspark/python_package/python_submit_yarn_client.sh.example

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
Looks like content in python_submit_yarn_client.sh.example is covered in README.md, do we need the script here? The same with python_submit_yarn_cluster.sh.example